### DISCUSSION TITLE
digital_IC_experiment/

### DISCUSSION THREAD

**Discussion by giscus** [trace1729/NCU-CS-BaoYan#9] [Q&A]
# digital_IC_experiment/

https://15947470421.github.io/digital_IC_experiment/

**Comment by wjnnone** [comment]
感谢学长

**Comment by HumbleExplorer** [comment]
一生一芯24.07讲义里面E5部分有对数字芯片设计流程的详细介绍，感兴趣的同学也可以了解一下。
